fiftyone/server/routes/embeddings_v2.py

```
@@ -0,0 +1,755 @@
+    ("/embeddings/v2/masks", EmbeddingsV2Masks),
+    ("/embeddings/v2/lasso-stage", EmbeddingsV2LassoStage),
+    ("/embeddings/v2/sample-info", EmbeddingsV2SampleInfo),
+]
+
+# Color bodies are ~1-2 MB at 500K points and a session of color-by
+# switching revisits few fields, so a small cap covers it
+_COLOR_CACHE_MAX = 8
+_color_cache = OrderedDict()
+
+
+def _color_cache_get(key):
```

Ruff (0.15.21)

[warning] Missing return type annotation for private function `_color_cache_get`

(ANN202)